content/guidance/tcop10/index.html.md

```diff
@@ -99,11 +99,11 @@ If you are agreeing a contract of over £20 million, the supplier must provide t
 
 ### Storing and standardising your data
 
-The gGovernment’s [Data Standards Authority](https://www.gov.uk/government/groups/data-standards-authority) and the [Open Standards Board](https://www.gov.uk/government/groups/open-standards-board) have approved a number of [open standards for data](https://www.gov.uk/government/collections/open-standards-for-government-data-and-technology) for use by the government. Where possible and appropriate, you should use these standards to make it easier to analyse data and share it with other departments.
+The government’s [Data Standards Authority](https://www.gov.uk/government/groups/data-standards-authority) and the [Open Standards Board](https://www.gov.uk/government/groups/open-standards-board) have approved a number of [open standards for data](https://www.gov.uk/government/collections/open-standards-for-government-data-and-technology) for use by the government. Where possible and appropriate, you should use these standards to make it easier to analyse data and share it with other departments.
```

Comment:
Random question - should we be using `Government` or `government`?

Reply:
[GOV.UK style](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style) uses 'government' in all situations. Can take some getting used to!